plugins/hls-fourmolu-plugin/src/Ide/Plugin/Fourmolu.hs

```diff
@@ -1,9 +1,10 @@
+{-# LANGUAGE CPP                      #-}
+{-# LANGUAGE DataKinds                #-}
```

Comment:
This extension is reshuffled because I ran `stylish-haskell` (which appears to be how this file is formatted) and it automatically did that change.

Same for the reordered imports below.

Reply:
So `stylish-haskell` isn't enforced by CI? If not, I'd be tempted to go back to `fourmolu`.

This file was formatted with `stylish-haskell` when it looked like that was becoming necessary (#693, #1439). But I lost track of where the decision on formatting this codebase ended up.

Reply:
We have a pre-commit hook that runs stylish-haskell, mentioned in the contributing guidelines. But it is not enforced by CI:

https://github.com/haskell/haskell-language-server/blob/7431cea783f3b92d327635c017dd5de87b4b8774/docs/contributing/contributing.md#L171-L214